src/main/java/botto/command/AddEventCommand.java

```diff
@@ -0,0 +1,62 @@
+package botto.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import botto.BottoException;
+import botto.task.Event;
+import botto.task.Task;
+import botto.util.Storage;
+import botto.util.TaskList;
+import botto.util.Ui;
+
+/**
+ * Command for adding an event
+ */
+public class AddEventCommand implements Command {
```

Comment:
I like how you had a Command interface where classes with similar behavior implement it!